Configuration/Generator/python/WM_14TeV_TuneCP5_cfi.py

```diff
@@ -11,11 +11,11 @@
                          maxEventsToPrint = cms.untracked.int32(0),
                          PythiaParameters = cms.PSet(
         pythia8CommonSettingsBlock,
-        pythia8CUEP8M1SettingsBlock,
+        pythia8CP5SettingsBlock,
         processParameters = cms.vstring(
             'WeakSingleBoson:ffbar2W = on',
             '24:onMode = off',
-            '24:onIfAny = 12 13',
+            '24:onIfAny = 13 14',
             ),
         parameterSets = cms.vstring('pythia8CommonSettings',
                                     'pythia8CUEP8M1Settings',
```

Comment:
this line also needs to be TuneCP5

Reply:
I think processParameters is missing here (see the other two configs)

Reply:
not sure I got this comment, sorry :(

Reply:
Mia picks mu nu_mu decays which makes sense as there is no W ->  nu_e mu in the SM.

Reply:
that is the intent of my update (while making this PR I noticed this "feature" .... am I messing up w/ the pdgID ? it should be right, now..... mu --> 13 and nu_mu --> 14, right ? 
![image](https://user-images.githubusercontent.com/4946419/116444379-f5d49100-a854-11eb-8e2e-250ebc06f70f.png)

Reply:
Right. I was just explaining. It is 100% correct.